docs/supported.md

```diff
@@ -0,0 +1,461 @@
+---
+id: supported
+title: Supported Functions
+---
+This is a list of TeX functions supported by KaTeX. It is sorted into logical groups.
+
+There is another version of this page, [with working examples](https://khan.github.io/KaTeX/function-support.html).
```

Comment:
I would probably just link to this page for now and then move this page to into the repo and style it appropriately.  Apparently docusaurus has a way to wrap HTML pages so that they look like they're part of the site.  https://docusaurus.io/docs/en/custom-pages

Reply:
Or as `Markdown` is superset of HTML, I think we can just copy its contents for now.

Reply:
Give me a couple of days before you take any drastic action.. I'll see what I can do with the docosaurus page. Either way, it will be good to have only one page to maintain.

Reply:
@ylemkimon I'd like the function support page to have tables styled differently than the GFM markdown tables now shown. For example, many of the tables should have no `<thead>` row.  I like your idea of copying at least some of the current function support page into this documentation.

If I add some classes to `website/static/static/index.css`, can I then use those class names to style elements in the `supported.md` file?

Reply:
Answering my own question: docusaurus docs [address this](https://docusaurus.io/docs/en/api-pages#using-static-assets).

Reply:
@ronkok `website/static/static/index.css` is for the main page(`index.html`), and for the styles needed in documentations, you can put them in the `website/static/css/custom.css` or another `.css` file in `website/static/css`.

Reply:
Can we leave a redirect link at https://khan.github.io/KaTeX/function-support.html that will forward to the new documentation? That existing page is fairly high in the search engine results for "katex".